narwhals/_compliant/any_namespace.py

```diff
@@ -53,8 +53,11 @@ def offset_by(self, by: str) -> CompliantT_co: ...
 
 class ListNamespace(_StoresCompliant[CompliantT_co], Protocol[CompliantT_co]):
     def len(self) -> CompliantT_co: ...
+
     def unique(self) -> CompliantT_co: ...
 
+    def get(self, index: int) -> CompliantT_co: ...
+
```

Comment:
Nitpick to keep it alphabetically sorted here as well

```suggestion
    def get(self, index: int) -> CompliantT_co: ...
    def len(self) -> CompliantT_co: ...
    def unique(self) -> CompliantT_co: ...
```